spec/requests/users_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    pending "add some examples (or delete) #{__FILE__}"
+  end
+end
```

Comment:
- [ ] Kindly make sure to implement tests for the `index` and `show` actions for the `user` controller.
- [ ] Please be mindful of the type of tests you'll need to implement for each of the actions (three tests per action), as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=If%20response%20status,correct%20placeholder%20text.) requirement.